dvc/command/experiments.py

```diff
@@ -274,14 +276,16 @@ def _sort(item):
     return ret
 
 
-def _format_time(timestamp):
-    if timestamp is None:
-        return FILL_VALUE
-    if timestamp.date() == date.today():
+def _format_time(datetime_obj, fill_value=FILL_VALUE, iso=False):
+    if datetime_obj is None:
+        return fill_value
+    if datetime_obj.date() == date.today():
         fmt = "%I:%M %p"
     else:
         fmt = "%b %d, %Y"
-    return timestamp.strftime(fmt)
+    if iso:
+        return datetime_obj.isoformat()
```

Comment:
Maybe let's do this after `if datetime_obj is None` branch, as it does not require `fmt`. It's a bit confusing right now.